test/text.js

```diff
@@ -6,6 +6,16 @@ var request = require('supertest')
 
 var bodyParser = require('..')
 
+/**
+ * @const
+ * whether current node version has brotli support
+ */
+var hasBrotliSupport = false
+try {
+  hasBrotliSupport = 'brotli' in require('process').versions
```

Comment:
The process module is just a global so no need to require it or have in a try catch.